tasks/libs/common/datadog_api.py

```diff
@@ -3,15 +3,21 @@
 from invoke.exceptions import Exit
 
 
-def create_metric(metric_type, metric_name, timestamp, value, tags, unit=None):
+def create_metric(metric_type, metric_name, timestamp, value, tags, unit=None, origin_metrics=None):
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏we sometimes use `metric_origin`, sometimes `origin_metric`. Maybe we should try to be consistent?